cli/azd/pkg/output/ux/title.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package ux
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/azure/azure-dev/cli/azd/pkg/output"
+)
+
+type MessageTitle struct {
+	Title     string
+	TitleNote string
+}
+
+func (t *MessageTitle) ToString(currentIndentation string) string {
+	if t.TitleNote != "" {
+		return fmt.Sprintf("\n%s\n%s\n",
+			output.WithBold(t.Title),
+			output.WithGrayFormat(t.TitleNote))
+	}
+	return fmt.Sprintf("\n%s\n", output.WithBold(t.Title))
+}
+
+func (t *MessageTitle) ToJson() []byte {
+	jsonBytes, _ := json.Marshal(output.EventForMessage(fmt.Sprintf("TITLE: %s, TITLE NOTE: %s", t.Title, t.TitleNote)))
```

Comment:
As @ellismg alluded to during the demo, I think we want a structured `type` here that isn't `consoleMessage`

Reply:
I think something closer to "OperationStart", and "OperationStop"